android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -1770,8 +1770,12 @@ private WritableMap createNetworkResponseInfo(final MaxNetworkResponseInfo respo
         WritableMap credentials = Arguments.createMap();
         for ( String key : credentialBundle.keySet() )
         {
-            String value = credentialBundle.getString( key, "" );
-            credentials.putString( key, value );
+            Object obj = credentialBundle.get( key );
+            if ( obj != null )
+            {
+                String value = obj.toString();
+                credentials.putString( key, value );
+            }
```

Comment:
I'd do something like this then, no need for non-String values in credentials.

```suggestion
            Object obj = credentialBundle.get( key );
            if ( obj instanceof String )
            {
                credentials.putString( key, (String) value );
            }
```

Reply:
Ok, thanks.   I updated and tested.  Please review again!